docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -555,6 +555,8 @@ To prevent unauthorized users from accessing data, this procedure is disabled by
 default. The procedure is enabled only when
 `iceberg.register-table-procedure.enabled` is set to `true`.
 
+In addition, the procedure expects registering same table with the same location, do not registering more than one table with same location.
```

Comment:
wrap into two lines.
Just curious how does the procedure register with same table(name)?

`do not registering` -> `avoids registering`